### PR TITLE
Update auth view tests for Django 2.0

### PR DESCRIPTION
**Pull request by benjaoming**
I also moved them to the `accounts` test module.

CC: @atombrella

### CODE REVIEW

File: tests/core/test_accounts.py [hunk 1]
```diff
@@ -57,3 +63,59 @@ def test_password_change(self):
...
+        self.client.force_login(self.superuser1)
+        response = self.client.get(wiki_settings.LOGIN_URL)
+        self.assertRedirects(response, reverse('wiki:root'))
+
+    def test_log_in(self):
+        self.client.post(
+            wiki_settings.LOGIN_URL,
+            {'username': SUPERUSER1_USERNAME, 'password': SUPERUSER1_PASSWORD}
+        )
+        self.assertTrue(self.superuser1.is_authenticated, True)
```
Comment (atombrella) [inline tests/core/test_accounts.py]: This should have been `self.assertIs`.

File: tests/core/test_accounts.py [hunk 2]
```diff
@@ -57,3 +63,59 @@ def test_password_change(self):
...
+        self.assertEqual(test_auth.email, 'test@test.com')
+
+
+@wiki_override_settings(ACCOUNT_HANDLING=True)
+class LogoutViewTests(RequireRootArticleMixin, DjangoClientTestBase):
+
+    def test_logout_account_handling(self):
+        self.client.get(wiki_settings.LOGOUT_URL)
+        user = auth.get_user(self.client)
+        self.assertFalse(auth.get_user(self.client).is_authenticated)
```
Comment (atombrella) [inline tests/core/test_accounts.py]: I think `self.assertIs` is the preferred style.

File: tests/core/test_accounts.py (hunk 1 above)
Comment (benjaoming) [reply tests/core/test_accounts.py]: This it cannot be... because it's either a boolean `True` (Django 2.0) or a `CallableBoolean(True)` (Django 1.11)

Comment (benjaoming) [reply tests/core/test_accounts.py]: Sorry should have added a comment, is done now

**Comment by benjaoming** [comment]
Welcome aboard the Core team, @atombrella :)

**Pull request merged**